schemas/context/destination-segment.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/destination-segment",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Destinations",
+  "type": "object",
+  "meta:extensible": false,
+  "meta:abstract": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/record"
+  ],
+  "description": "Destinations Segment Mapping Schema.",
+  "definitions": {
+    "destination-segment": {
+      "properties": {
+        "xdm:destinationID": {
+          "title": "Destination ID",
+          "type": "string",
+          "description": "Unique Identifier for destination flow"
+        },
+        "xdm:segmentID": {
+          "title": "Segment Identifier",
```

Comment:
To keep the usage of ID consistent across the entire schema defs, can be "Segment ID"